lib/lightning_web/controllers/user_auth.ex

```diff
@@ -190,6 +193,16 @@ defmodule LightningWeb.UserAuth do
     end
   end
 
+  defp get_current_resource(token) do
+    case token do
+      "prc_" <> _rest ->
+        Lightning.VersionControl.get_repo_connection_for_token(token)
+
+      _user_token ->
+        Accounts.get_user_by_api_token(token)
+    end
```

Comment:
Hmm I'm not in love with the `prc_` prefix on all tokens, right? It's `prc_<JWT>` hey?